vscode/oso/README.md

```diff
@@ -6,7 +6,10 @@ Syntax highlighting for `.polar` files as used by [Oso](https://www.osohq.com).
 
 ## Language Server Protocol (LSP) Functionality
```

Comment:
Not sure that users care that this is using LSP protocol.

Reply:
1369607a